xxe/xxe.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/Contrast-Security-OSS/go-test-bench/utils"
 )
 
+// FIXME: These don't appear to be XXE vulnerabilities to me... Can we delete this package?
```

Comment:
This is a question for reviewers. I think it's best to just remove this route handler so that the "underConstruction" page is used instead, because it doesn't seem to be using XXE at all from what I can tell.

Reply:
The team agreed that we should remove this package but that work should be split into a separate PR. This will be internally tracked by GO-1041 .